ai/worker/runner.gen.go

```diff
@@ -461,6 +461,15 @@ type ValidationError_Loc_Item struct {
 	union json.RawMessage
 }
 
+// Version defines model for Version.
+type Version struct {
+	ModelId  string `json:"model_id"`
+	Pipeline string `json:"pipeline"`
```

Comment:
`Pipeline` here is name of pipeline right? Should we also add `PipelineId`?

Reply:
No, it's the pipeline name. Not the ID. So, it's like `live-video-to-video`. We call it everywhere in the code "pipeline", so I think it's ok to have like this.